### PR TITLE
Feature/input examples

### PR DESCRIPTION
**Pull request by kyleturco**
Adding input examples to the doc site

### CODE REVIEW

File: docs/app/Examples/elements/Input/Variations/Size.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+          <i className='search icon' />
+        </Input>
+        <br />
+        <Input className='massive icon' placeholder='Search...'>
+          <i className='search icon' />
+        </Input>
+      </div>
+    );
+  }
+}
```
Comment (kyleturco) [inline docs/app/Examples/elements/Input/Variations/Size.js]: Wasn't sure the best way to handle this example. I put all the size examples in one file with `br` between them. Alternate ideas would be create separate files.. eg.. `SizeMini.js`, `SizeBig.js` and then put them all in the Size example section. Thought it was 50/50 on adding lots of files vs putting all this in one. Open to suggestions

File: docs/app/Examples/elements/Input/Variations/Variations.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+          examplePath='elements/Input/Variations/Inverted'
+        />
+        <ComponentExample
+          title='Fluid'
+          description='An input can take on the size of its container'
+          examplePath='elements/Input/Variations/Fluid'
+        />
+        <ComponentExample
+          title='Size'
+          description='An input can vary in sizee'
```
Comment (kyleturco) [inline docs/app/Examples/elements/Input/Variations/Variations.js]: I'll fix the "sizee" typo after all the other comments are

File: docs/app/Examples/elements/Input/Variations/Labeled.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import React, {Component} from 'react';
+import {Input} from 'stardust';
+
+export default class Labeled extends Component {
+  render() {
+    return (
+      <Input className='labeled' placeholder='mysite.com'>
+        <div className='ui label'>
```
Comment (kyleturco) [inline docs/app/Examples/elements/Input/Variations/Labeled.js]: See Issue #61 - currently Input can't handle a label. This is causing it to render weird where the button can't go on the left side.

File: docs/app/Examples/elements/Input/Variations/Fluid.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import React, {Component} from 'react';
+import {Input} from 'stardust';
+
+export default class Fluid extends Component {
+  render() {
+    return (
+      <Input className='fluid icon' placeholder='Search...'>
+        <i className='search icon' />
+      </Input>
```
Comment (kyleturco) [inline docs/app/Examples/elements/Input/Variations/Fluid.js]: Same issue as the "Labeled.js". Button is only displaying to the right of the input field

Comment (levithomason) [inline docs/app/Examples/elements/Input/Variations/Fluid.js]: fixed

**Pull request merged**